posthog/settings/sentry.py

```diff
@@ -109,7 +108,10 @@ def traces_sampler(sampling_context: dict) -> float:
         task = sampling_context.get("celery_job", {}).get("task")
         if task == "posthog.celery.redis_heartbeat":
             return 0.0001  # 0.01%
-        if task == "posthog.celery.redis_celery_queue_depth":
+        if (
+            task == "posthog.celery.redis_celery_queue_depth"
+            or task == "posthog.celery.redis_celery_queue_depth_usage_reports"
+        ):
```

Comment:
Can also now write `if task in ("...", "...")` for better readability.

Reply:
Updated for this